<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Timetable</class>
 <widget class="QWidget" name="Timetable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>630</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>841</width>
    <height>630</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>841</width>
    <height>630</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Timetable</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color: rgb(228, 235, 243)
}

QMessageBox {
    background-color: rgb(255, 255, 255);
	font: 57 10pt &quot;Uniform Rnd Medium&quot;;
}

QMessageBox::ok {
    background-color:rgb(233, 66, 146);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: rgb(255, 255, 255);
	color: rgb(255, 255, 255);
	font: 57 10pt &quot;Uniform Rnd Medium&quot;;
}

</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QPushButton" name="HomeButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>530</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">    background-color: rgb(255, 255, 255);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: rgb(255, 255, 255);
	color: rgb(68, 68, 68);
	font: 57 10pt &quot;Uniform Rnd Medium&quot;;
</string>
   </property>
   <property name="text">
    <string>Show current date</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AddEventButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>530</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Uniform Rnd Medium</family>
     <pointsize>10</pointsize>
     <weight>7</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">    background-color: rgb(255, 255, 255);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: rgb(255, 255, 255);
	font: 57 10pt &quot;Uniform Rnd Medium&quot;;
	color: rgb(68, 68, 68);
</string>
   </property>
   <property name="text">
    <string>Add Event</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>0</y>
     <width>331</width>
     <height>631</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255)</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QLabel" name="DateLabel">
       <property name="font">
        <font>
         <family>Uniform Rnd Medium</family>
         <pointsize>28</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(68, 158, 218)</string>
       </property>
       <property name="text">
        <string>WEDNESDAY, 24TH DECEMBER</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="indent">
        <number>-1</number>
       </property>
      </widget>
      <widget class="QListWidget" name="EventListWidget">
       <property name="font">
        <font>
         <family>Comfortaa</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"> QScrollBar:vertical {
            border: 0px solid #999999;
            background:white;
            width:10px;    
            margin: 0px 0px 0px 0px;
}

 QScrollBar::handle:vertical {         
            min-height: 0px;
          	border: 0px solid red;
			border-radius: 4px;
			background-color: black;
}

 QScrollBar::add-line:vertical {       
            height: 0px;
            subcontrol-position: bottom;
            subcontrol-origin: margin;
}

 QScrollBar::sub-line:vertical {
            height: 0 px;
            subcontrol-position: top;
            subcontrol-origin: margin;
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustIgnored</enum>
       </property>
       <property name="autoScroll">
        <bool>false</bool>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="isWrapping" stdset="0">
        <bool>false</bool>
       </property>
       <property name="sortingEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>431</width>
     <height>431</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QCalendarWidget{
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: rgb(255, 255, 255);
}

QCalendarWidget QToolButton {
  	height: 60px;
  	width: 150px;
  	color: rgb(233, 66, 146);
	font: 24pt &quot;Comfortaa&quot;;
  	font-size: 24px;
	background: url(qrc:/ui/next.png)t;
  	icon-size: 10px, 10px;
  	background-color: rgb(255, 255, 255);
  }

  QCalendarWidget QMenu {
	color: rgb(233, 66, 146);
	background-color: rgb(255, 255, 255);
	font: 24pt &quot;Comfortaa&quot;;
}

QCalendarWidget QSpinBox { 
	font: 57 24pt &quot;Uniform Rnd Medium&quot;;
  	width: 150px; 
  	color: rgb(233, 66, 146); 
  	background-color: rgb(255, 255, 255); 
  	selection-background-color: rgb(136, 136, 136);
  	selection-color: rgb(255, 255, 255);
  }

QCalendarWidget QSpinBox ::up-button{ 
    subcontrol-origin: border;
    subcontrol-position: top right; 
	background-color: white;
	width: 20px;
	border: 0px;
}

QCalendarWidget QSpinBox ::up-arrow{ 
	image: url(:/ui/down.png);
	width: 10px;
	height: 10px;
}

  /* header row */
  QCalendarWidget QWidget 
	{ 
	font: 57 16pt &quot;Uniform Rnd Medium&quot;;
	alternate-background-color: rgb(255, 255, 255);
	}
   
  /* normal days */
  QCalendarWidget QAbstractItemView:enabled 
  {
	font: 57 16pt &quot;Uniform Rnd Medium&quot;;
  	font-size:16px;  
  	color: rgb(63, 63, 63);
  	background-color: rgb(255, 255, 255);  
  	selection-background-color: rgb(84, 163, 219); 
  	selection-color: rgb(255, 255, 255); 
  }

QCalendarWidget QAbstractItemView::item  {
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: rgb(255, 255, 255); 
}

QCalendarWidget QAbstractItemView:disabled 
{ 
color: rgb(247, 247, 247); 
}
</string>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="firstDayOfWeek">
    <enum>Qt::Monday</enum>
   </property>
   <property name="gridVisible">
    <bool>false</bool>
   </property>
   <property name="horizontalHeaderFormat">
    <enum>QCalendarWidget::SingleLetterDayNames</enum>
   </property>
   <property name="verticalHeaderFormat">
    <enum>QCalendarWidget::NoVerticalHeader</enum>
   </property>
   <property name="navigationBarVisible">
    <bool>true</bool>
   </property>
   <property name="dateEditEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>451</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">    background-color: rgb(255, 255, 255);
    border-style: outset;
    border-width: 2px;
    border-radius: 20px;
    border-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <zorder>HomeButton</zorder>
  <zorder>AddEventButton</zorder>
  <zorder>widget_2</zorder>
  <zorder>widget</zorder>
  <zorder>calendarWidget</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
